src/guides/v2.3/frontend-dev-guide/tools/using_grunt.md

```diff
@@ -111,10 +111,10 @@ This path is also added to your .gitignore by default.
 
 You can also use the other way to declare a custom config file:
 
-1. In your Grunt related scripts, in the file router set the alias and path to the default configuration file. For example, for the `themes.js` configuration file this would look like following:
+1. In your Grunt related scripts, in the file router set the alias and path to the custom configuration file. For example, for set the custom `themes.loc.js` configuration file this would look like following:
```

Comment:
```suggestion
1. In your Grunt related scripts, in the file router, set the alias and path to the custom configuration file. For example, to set the custom `themes.loc.js` configuration file, this would look like following:
```